### ISSUE TITLE
Bug: Cannot reach last slide using navigation button

### ISSUE THREAD

**Issue by dfvc** [SSENSE/vue-carousel#345]
## Bug Report

**Current Behavior**
Cannot reach the last slides by swiping.

**Input Code and steps to reproduce**

```js
<carousel
          :per-page="2"
          :scroll-per-page="false"
          :navigation-enabled="true"
          :pagination-enabled="false"
>
...
</carousel>
```

**Expected behavior/code**
Before 0.16.0, it was not possible to reach the last slide by using navigation "next" button. Swiping everything worked fine. Now, after update to 0.16.0, this behavor is inverted, i.e., I can reach the last slider by pressing "next" button but not by swiping.

**Comment by jorgemurta** [comment]
Having the same issue @SEE

**Comment by jorgemurta** [comment]
This can be reproduced on the "Navigation buttons and scrollPerPage false" scenario on the project's Vue Play

**Comment by jorgemurta** [comment]
@dfvc Thank you!

**Comment by quinnlangille** [comment]
Hmm thanks for flagging guys, I can reproduce on my end. The last minor added some changes to scrolling logic, we'll have to investigate to see where the issue is. Clearly our fix didn't fully resolve! I'll mark as a bug - if anyone fixes in the mean time, feel free to make a pr!

**Comment by dfvc** [comment]
@quinnlangille, there's already a pr for it: https://github.com/SSENSE/vue-carousel/pull/346

**Comment by jorgemurta** [comment]
@quinnlangille there is already https://github.com/SSENSE/vue-carousel/pull/346

**Comment by quinnlangille** [comment]
Ah, didn't see (still going thru notifications for the week and always do issues first). I'll review and continue conversation on pr - thanks for pointing it out :octocat:

**Comment by josephting** [comment]
This should be closed now with #346 merged. 💯

**Comment by rfriberg** [comment]
Hi! I'm still seeing evidence of this behavior.  I.e., I can reach the last slide by pressing "next" button but not by swiping.

I was able to reproduce on that same "Navigation buttons and scrollPerPage false" scenario mentioned above with some minor changes:  load 9 or more slides and change `perPage` to anything more than '2':

```
const images = [
  "https://res.cloudinary.com/ssenseweb/image/upload/b_white,c_lpad,g_south,h_1086,w_724/c_scale,h_560/v588/171924M176006_1.jpg",
  "https://res.cloudinary.com/ssenseweb/image/upload/b_white,c_lpad,g_south,h_1086,w_724/c_scale,h_560/v588/171924M176005_2.jpg",
  "https://res.cloudinary.com/ssenseweb/image/upload/b_white,c_lpad,g_south,h_1086,w_724/c_scale,h_560/v588/171924M176003_3.jpg",
  "https://res.cloudinary.com/ssenseweb/image/upload/b_white,c_lpad,g_south,h_1086,w_724/c_scale,h_560/v588/171924M176004_4.jpg",
  "https://res.cloudinary.com/ssenseweb/image/upload/b_white,c_lpad,g_south,h_1086,w_724/c_scale,h_560/v588/171924M176002_5.jpg",
  "https://res.cloudinary.com/ssenseweb/image/upload/b_white,c_lpad,g_south,h_1086,w_724/c_scale,h_560/v588/171924M176001_2.jpg",
  "https://res.cloudinary.com/ssenseweb/image/upload/b_white,c_lpad,g_south,h_1086,w_724/c_scale,h_560/v588/171924M176006_1.jpg",
  "https://res.cloudinary.com/ssenseweb/image/upload/b_white,c_lpad,g_south,h_1086,w_724/c_scale,h_560/v588/171924M176005_2.jpg",
  "https://res.cloudinary.com/ssenseweb/image/upload/b_white,c_lpad,g_south,h_1086,w_724/c_scale,h_560/v588/171924M176003_3.jpg"
]
...
{ props: { navigationEnabled: true, scrollPerPage: false, perPage: 3 } }
```
